modern/.env

```diff
@@ -1,4 +1,5 @@
 CONAN_VERSION=1.37.2
+DOCKER_TAG=1.37.2
```

Comment:
Why separate DOCKER_TAG and CONAN_VERSION?

Reply:
I need to tag the images built from PRs with the corresponding commit, so they are not used by other jobs running other branches/pull requests. Only images generated in `master` can use the tag with the Conan version.